projects/legacy/components/input-tag/input-tag.component.ts

```diff
@@ -113,6 +114,9 @@ export class TuiInputTagComponent
     @ContentChild(TuiDataListDirective, {read: TemplateRef})
     protected readonly datalist?: TemplateRef<TuiContext<TuiActiveZone>>;
 
+    @ContentChild(PolymorpheusOutlet, {read: ElementRef})
+    protected readonly valueContent?: ElementRef<HTMLSpanElement>;
```

Comment:
Type is wrong, just use `PolymorpheusOutlet` and type can be `unknown`. Outlet is a structural directive, it does not have an `ElementRef`, not sure how that works, probably it reaches up for closest `ElementRef`. If you would ask me this question on an interview I'd say it would be `null` :)

Reply:
Still, it gets to the ElementRef, but not the span element with the text, but to some kind of artifact from the directive, interestingly

is it better to just change to `unknown` type or, for clarity, hang some kind of tag on the content wrapper and not get tied to PolymerphousOutlet?

![image](https://github.com/user-attachments/assets/a740c0ac-ca05-4302-8126-22f7fe2a2ae9)